packages/shared/src/components/sidebar/SquadButton.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { ReactElement } from 'react';
+import Link from 'next/link';
+import classNames from 'classnames';
+import { ClickableNavItem } from './ClickableNavItem';
+import { ItemInner, ListIcon, NavItem, SidebarMenuItem } from './common';
+import SquadIcon from '../icons/Squad';
+import { SquadVersion } from '../../lib/featureValues';
+import AddUserIcon from '../icons/AddUser';
+import { Button } from '../buttons/Button';
+import PlusIcon from '../icons/Plus';
+
+interface SquadButtonProps {
+  sidebarRendered?: boolean;
+  sidebarExpanded: boolean;
+  squadVersion: SquadVersion;
+  squadButton: string;
+  squadForm: string;
+  onSquadClick: () => void;
+}
+
+function SquadButton({
```

Comment:
The link should open in a new tab

Reply:
Ah it was mentioned in the kickoff to have it in the same tab, but will adjust :)